migrations/20240930203726_add_web_authn.up.sql

```diff
@@ -0,0 +1,3 @@
+alter table {{ index .Options "Namespace" }}.mfa_factors add column if not exists web_authn_credential jsonb null;
+alter table {{ index .Options "Namespace" }}.mfa_factors add column if not exists aaguid uuid null;
```

Comment:
Wonder if this would be more useful like:

```suggestion
alter table {{ index .Options "Namespace" }}.mfa_factors add column if not exists web_authn_aaguid uuid null;
```